DockDoor/Utilities/Window Management/WindowUtil.swift

```diff
@@ -714,9 +722,10 @@ enum WindowUtil {
             {
                 let updatedWindow = existingWindow
                 await Task.detached(priority: .userInitiated) {
-                    if let image = try? await captureWindowImage(window: actualSCWindow, forceRefresh: true) {
+                    if let (image, imageDate) = try? await captureWindowImage(window: actualSCWindow, forceRefresh: true) {
```

Comment:
I'm new to swift. Does this if-statement still work correctly?

Reply:
Yes this works